python/distrdf/dask/test_reducer_merge.py

```diff
@@ -308,6 +338,36 @@ def test_distributed_snapshot_columnlist(self):
         for filename in tmp_files:
             os.remove(filename)
 
+    def test_distributed_snapshot_lazy(self):
+        """Test that `Snapshot` can be still called lazily in distributed mode"""
+        # A simple dataframe with ten sequential numbers from 0 to 9
+        df = Dask.RDataFrame(10, daskclient=self.client)
+
```

Comment:
Same comment as above for `test_distributed_snapshot` and `test_distributed_snapshot_lazy`.